web/src/components/nav/switch.js

```diff
@@ -39,4 +44,29 @@ export const routes = [
     exact: true,
     component: About,
   },
+  {
+    path: "/login",
```

Comment:
We may want to adjust these routes. I'm thinking that we may want to preface the account-related accounts with `account`. We may also be able to combine workflows into a single route with a component that holds both states
For example:

- `/account/auth` Handles login and logout with an "accountAuthentication" component
- `/account/password` Handles the reset and confirmation within a  "accountPassword" component
  - uid and token are held in the component's state, avoiding the need to put it in the url
- `/account` Can grow into an account management console page. But for now, if the user does not have an account, they can create one here. It would combine "register" and "activate" into a single component.

Reply:
For "activate", I need to keep the path as `/activate/:uid/:token`. I can change for the other ones.

Reply:
I'll address this comment/push up additional changes once we're on the same page about this one.

Reply:
Yeah, on second thought, I don't think I like my recommendation.